src/user/emails.go

```diff
@@ -43,4 +43,15 @@ If it was you, you can click the following link and fill in the form: <br>
 <a class="button" href="%s">Reset Password</a> <br><br>
 See you soon!! <br>
 `, nickname, link))
-}
\ No newline at end of file
+}
+
+func SendLoginNotificationEmail(nickname string, email string) error {
+	return utils.SendEmail(
+		[]string{email},
+		"Cosmos Login Notification",
+		fmt.Sprintf(`<h1>Login Notification</h1>
+Hello %s, <br>
+Your account has been logged into. If it wasn't you, please reset your password and alert your server admin. <br>
+If it was you, you can ignore this email. <br><br>
+`, nickname))
+}
```

Comment:
Would love to get your opinions on what should we keep the template as? What field do we wish to  show in the email  
@azukaar

Reply:
Would be good to add the date and IP to the info, other than that this is good